libs/client-sdk/include/launchdarkly/client_side/bindings/c/config/builder.h

```diff
@@ -0,0 +1,487 @@
+/** @file */
```

Comment:
This is mostly moving the old `builder.h` to this new location. Log related config was removed and remains in common.